### ISSUE TITLE
Blank pronunciations included in pronunciation lists

### ISSUE THREAD

**Issue by peldas** [peldas/yomitan-dicts#1]
Some dictionaries have blank pronunciations which should be skipped on parsing.
![image](https://github.com/peldas/yomitan-dicts/assets/13115959/2912fb78-a6bc-40d3-aad2-d2c37d7c63c9)

**Comment by peldas** [comment]
Workaround for card creation – use these custom handlebars to grab the first non-blank pronunciation:
```handlebars
{{#*inline "phonetic-transcription-first"}}
    {{~set "reading" definition.reading~}}
    {{~#if (op ">" definition.phoneticTranscriptions.length 0)~}}
        {{~set "any" false~}}
        {{~#each definition.phoneticTranscriptions~}}
                {{~#each phoneticTranscriptions~}}
                    {{~#unless (get "any")}}
                        {{~#if (op ">" ipa.length 0)~}}
                            {{ipa~}}{{~set "any" true~}}
                        {{~/if~}}
                    {{~/unless~}}
                {{~/each~}}
        {{~/each~}}
    {{~/if~}}
{{/inline}}
```

```handlebars
{{#*inline "phonetic-transcription-first-if-not-equal-to-reading"}}
    {{~set "reading" definition.reading~}}
    {{~#if (op ">" definition.phoneticTranscriptions.length 0)~}}
        {{~set "any" false~}}
        {{~#each definition.phoneticTranscriptions~}}
                {{~#each phoneticTranscriptions~}}
                    {{~#unless (get "any")}}
                        {{~#if (op "&&" (op ">" ipa.length 0) (op "!=" ipa (get "reading")))~}}
                            {{ipa~}}{{~set "any" true~}}
                        {{~/if~}}
                    {{~/unless~}}
                {{~/each~}}
        {{~/each~}}
    {{~/if~}}
{{/inline}}
```

**Comment by peldas** [comment]
Fixed in 335ad7d8d7602b660cab08024fb94d8aae7490f6.
